src/physics/arcade/Body.js

```diff
@@ -1899,7 +1899,7 @@ var Body = new Class({
      * @since 3.0.0
      *
      * @param {number} x - The horizontal bounce, relative to 1.
-     * @param {number} y - The vertical bounce, relative to 1.
+     * @param {?number} y - The vertical bounce, relative to 1.
```

Comment:
Thank you for your PR. The way to define is a parameter is optional is to put the parameter name in square braces. So this line should be: `@param {number} [y=x] - The vertical bounce, relative to 1.`